src/oc_erchef/apps/chef_objects/src/chef_node.erl

```diff
@@ -76,12 +77,16 @@
           {<<"override">>, chef_json_validator:attribute_spec()},
           {<<"automatic">>, chef_json_validator:attribute_spec()},
 
-          {<<"run_list">>, chef_json_validator:run_list_spec()}
+          {<<"run_list">>, chef_json_validator:run_list_spec()},
+
+          {{opt, <<"policy_name">> }, {string_match, chef_regex:regex_for(policy_file_name)}},
+          {{opt, <<"policy_group">> }, {string_match, chef_regex:regex_for(policy_file_name)}}
          ]}).
 
 -define(VALID_KEYS,
         [<<"name">>, <<"chef_environment">>, <<"json_class">>, <<"chef_type">>,
-         <<"normal">>, <<"default">>, <<"override">>, <<"automatic">>, <<"run_list">>]).
+         <<"normal">>, <<"default">>, <<"override">>, <<"automatic">>, <<"run_list">>,
+         <<"policy_name">>, <<"policy_group">> ]).
 
 -define(create_if_missing_fields,
         [{<<"chef_environment">>, <<"_default">>},
```

Comment:
I don't think there is a sane default. I was planning on having `null` for these fields be an indication to `chef-client` that it's running in "not policyfiles mode". The only thing I think I could put as a default is something that would be invalid for the user to upload, like `!nothing` but that leads to a lot of ugly elsewhere, I think.